### PR TITLE
Slightly improved error message when pushing a image to the library that already exist

### LINKED ISSUE #3408: confusing output when pushing existing image to library
## Version of Singularity:

3.2.0rc2

```
[vagrant@localhost tmp]$ singularity push -U illolcow.sif library://jscook2345-double/default/illolcow.sif
WARNING: Skipping container verifying
INFO:    Image is already present in the library - not uploading.
INFO:    Setting tag latest
WARNING: latest replaces an existing tag
```

A few things:

* I would figure that this would upload the image no matter what, but it appears if the image exists it will not overwrite it.

* There is no `--force` option to overwrite, so perhaps there should be?

* Since the image is not being uploaded, why is there additional output after that message?

* I'm not sure INFO is right for saying 'we are not uploading your image'.  Perhaps just a message than exit?

### PR DESCRIPTION
**Pull request by WestleyK**
## Description of the Pull Request (PR):

This PR will slightly improve the warning message when pushing a container to the library that already exist.

Example:

```bash
$ singularity push -U test.sif library://westleyk/foo/bar:latest,1.0
WARNING: Skipping container verifying
INFO:    Image already exist in the library
INFO:    Setting tag 1.0
```

## This fixes or addresses the following GitHub issues:

- Fixes #3408 

<br>

### CODE REVIEW

File: pkg/client/library/push.go [hunk 1]
```diff
@@ -95,7 +95,7 @@ func UploadImage(filePath string, libraryRef string, libraryURL string, authToke
 		}
 		sylog.Debugf("Upload completed OK\n")
 	} else {
-		sylog.Infof("Image is already present in the library - not uploading.\n")
+		return fmt.Errorf("image (%q) is already present in the library: not uploading", filePath)
```
Comment (jmstover) [inline pkg/client/library/push.go]: This isn't an error that should be called with `Errorf` should it? I mean, if you give it the same image, but a different tag, the image isn't uploaded, but the tag is created.

**Review by jmstover — CHANGES_REQUESTED** [review]

File: pkg/client/library/push.go (hunk 1 above)
Comment (WestleyK) [reply pkg/client/library/push.go]: Oh yeah, Let me see if I can fix this up... :+1:

Comment (WestleyK) [reply pkg/client/library/push.go]: @jmstover, Better now?

Comment (jmstover) [reply pkg/client/library/push.go]: Hrmm... This is where other folks need to chime in (@jscook2345 ??). I think a `Infof is perfectly acceptable here, and ... it isn't really a warning. It's just an informational message about the upload...

Comment (jmstover) [reply pkg/client/library/push.go]: And that was modified in between me starting to write this and actually finishing it... :P

**Review by jmstover — APPROVED** [review]

**Pull request merged**